.distignore

```diff
@@ -0,0 +1,9 @@
+/.wordpress-org
```

Comment:
Note that I previously added a `.gitattributes` that ignores some of these: https://github.com/10up/action-wordpress-plugin-deploy/tree/develop?tab=readme-ov-file#gitattributes

I don't have a strong preference on which we use, but do you mind merging the two?

Reply:
indeed, my mistake. will remove this file

Reply:
done c4c2413b6a848cc0ead39c9768a84e1d073d7596